src/components/ChartSlot.vue

```diff
@@ -189,3 +217,25 @@
     })
 </script>
 
+<style lang="scss">
+    .chart-container {
+        overflow-y: scroll;
+        width: 100%;
+    }
+
+    .chart {
+        width: 100%;
+    }
+
+    .annotation-text {
```

Comment:
Plotly doesn't really provide a nice way to change font weight on annotations in config, so doing it through css here, though it should really be part of the chart configuration